rpmio/rpmio.cc

```diff
@@ -462,6 +462,7 @@ static FD_t urlOpen(const char * url, int flags, mode_t mode)
 	unlink(dest);
     } else {
 	fd = NULL;
+	errno = ENOENT;
```

Comment:
Can you explain why this is needed a bit more? Also, the commit message says "urlOpen: Set errno to ENOTDIR on error", shouldn't it say ENOENT instead?